### PR TITLE
Feat: Improve text for NF commitment

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Improve the text that explains the new Neurons' Fund commitment.

# Changes

* Edit the value of an i18n key.

# Tests

Not necessary.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary. Not seen by any user yet.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3687** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/3687" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/3687?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/lib/i18n/en.json [hunk 1]
```diff
@@ -748,7 +748,7 @@
     "max_user_commitment_reached": "Maximum commitment reached",
     "not_eligible_to_participate": "You are not eligible to participate in this swap.",
     "getting_sns_open_ticket": "We're connecting with the SNS swap. This might take more than a minute.",
-    "current_nf_commitment_description": "The NF commitment increases based on direct participation. <a href=\"https://internetcomputer.org/docs/current/tokenomics/nns/neurons-fund\" rel=\"noopener noreferrer\" aria-label=\"more info about neurons' fund\" target=\"_blank\">Learn more</a>.",
+    "current_nf_commitment_description": "The NF commitment increases based on direct participation. The exact amount is computed at the end of the swap. <a href=\"https://internetcomputer.org/docs/current/tokenomics/nns/neurons-fund\" rel=\"noopener noreferrer\" aria-label=\"more info about neurons' fund\" target=\"_blank\">Learn more</a>.",
```
Comment (dskloetd) [inline frontend/src/lib/i18n/en.json:751]: Do we expect users to know what "NF" is or should that say "neurons' fund"?

**Review by dskloetd — APPROVED** [review]

File: frontend/src/lib/i18n/en.json (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/i18n/en.json]: It appears right below the text "Neurons' Fund Commitment". And it's already long, that's why I did it with NF. But it's true we never use this NF anywhere else...

I changed it.

**Pull request merged**